README.md

```diff
@@ -1,29 +1,18 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+# My Personal Portfolio
 
-# Project Name
+My portfolio where I show to the world what can i offer them.
 
-> One paragraph statement about the project.
+![screenshot](./app_portfolio_main.png)
 
-![screenshot](./app_screenshot.png)
-
-Additional description about the project and its features.
+In this project I show my skills and my other projects and is a way to centralize all my projects.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo
-
-[Live Demo Link](https://livedemo.com)
-
+- HTML5
+- CSS3
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
 
 To get a local copy up and running follow these simple example steps.
```

Comment:
- [ ] Please complete the instructions here on how users can get a local copy of your project